torch/utils/flop_counter.py

```diff
@@ -345,6 +434,9 @@ def _create_pre_module(self, name):
         class PopState(torch.autograd.Function):
             @staticmethod
             def forward(ctx, *args):
+                if self.in_backward:
+                    self.parents = ["Global"]
+                    self.in_backward = True
```

Comment:
What's going on here? I get the changes to the conv flop formula, but not what else is happening

Reply:
There was another bug related to clearing of the forwards/backwards stack.

The problem here is that when we have `mod(x)`, we insert an autograd.Function that runs before we enter the module and after we exit the module. Then, in the backwards pass, the corresponding backwards of the autograd.Function runs.

The problem is that if the input does not require grad, it's possible that the backward never runs.